### DISCUSSION TITLE
Improve create a plan UX

### DISCUSSION THREAD

**Discussion by Lpaydat** [subquery/subql#1025] [Frontier TestNet - General]
Many people are baffled about how to create a default plan because the default one is simply an empty item, so many will think they must select a project from the list and are unaware that this field can be left blank.

Instead of an empty item, I believe we should use a placeholder with a different style (for example, _Default Plan_).

**Comment by heyacherry** [comment]
Thank you for your suggestions!
@lilian79511 please has a review on this feature.
